components/dashboard/src/settings/Integrations.tsx

```diff
@@ -580,63 +612,61 @@ function GitIntegrationModal(props: ({
         }
     };
 
-    return (<Modal visible={!!props} onClose={close}>
-        <h3 className="pb-2">{props.mode === "new" ? "New Git Integration" : "Git Integration"}</h3>
+    return (<Modal visible={!!props} onClose={close} closeable={props.closeable}>
+        <h3 className="pb-2">{mode === "new" ? "New Git Integration" : "Git Integration"}</h3>
         <div className="space-y-4 border-t border-b border-gray-200 dark:border-gray-800 mt-2 -mx-6 px-6 py-4">
-            {props.mode === "edit" && props.provider.status === "pending" && (
+            {mode === "edit" && providerEntry?.status !== "verified" && (
                 <div className="flex rounded-md bg-gitpod-kumquat-light p-3">
                     <img className="w-4 h-4 mx-2 my-auto" src={exclamation} />
                     <span className="text-red-600 dark:text-red-400 hover:text-red-800 dark:hover:text-red-300">You need to activate this integration.</span>
                 </div>
             )}
             <div className="flex flex-col">
-                <span className="text-gray-500">Configure a git integration with a GitLab or GitHub self-hosted instance.</span>
+                <span className="text-gray-500">{props.headerText || "Configure a git integration with a GitLab or GitHub self-hosted instance."}</span>
             </div>
-            {props.mode === "new" && (
+
+            <div className="overscroll-contain max-h-96 overflow-y-auto">
```

Comment:
**suggestion**: What do you think of adding a small right padding here to avoid collapsing the scroll bar on top of the inoput elements? It did [something similar](https://github.com/gitpod-io/gitpod/blob/ce05f58c44d9d8ff4cf2a78e23a31b7571d42454/components/dashboard/src/workspaces/StartWorkspaceModal.tsx#L61) with the New Workspace modal (Recent/Examples).

```suggestion
            <div className="overscroll-contain max-h-96 overflow-y-auto pr-2">
```